[nnc][quantization] quantized ops for BI bytedoc via aten

IvanKobzarev opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #68790
* #68736

Differential Revision: [D32609427](https://our.internmc.facebook.com/intern/diff/D32609427)

IvanKobzarev committed:
[nnc][quantization] quantized sigmoid via aten

[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).

ZolotukhinM commented on torch/csrc/jit/tensorexpr/external_functions.cpp:
```diff
@@ -249,6 +256,7 @@ void nnc_aten_quantized_conv2d_relu(
 }
 
 #ifndef _WIN32
+
```
Nit: unnecessary change

ZolotukhinM reviewed (approved):
👍

IvanKobzarev committed:
Update on "[nnc][quantization] quantized sigmoid via aten"


Differential Revision: [D32609427](https://our.internmc.facebook.com/intern/diff/D32609427)

[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).

IvanKobzarev committed:
Update on "[nnc][quantization] quantized sigmoid via aten"


Differential Revision: [D32609427](https://our.internmc.facebook.com/intern/diff/D32609427)

[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).

IvanKobzarev committed:
Update on "[nnc][quantization] quantized ops for BI bytedoc via aten"


Differential Revision: [D32609427](https://our.internmc.facebook.com/intern/diff/D32609427)

[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).
